app/views/users/edit.html.haml

```diff
@@ -1,22 +1,31 @@
-.p-8
-  %div.my-8
-    %h2.h3 Auth tokens (advanced feature; safe to ignore)
-    - current_user.auth_tokens.each do |auth_token|
-      %div.mt-2
-        = form_with(model: auth_token, class: 'inline') do |form|
-          %span
-            = form.label :secret, 'Secret (required)'
-            = form.text_field :secret
+- content_for(:page_assets) do
+  = ts_tag('ujs')
 
-          %span.ml-8
-            = form.label :name, 'Name (optional)'
-            = form.text_field :name
+%div.my-8
```

Comment:
I suggest viewing the diff with whitespace changes ignored: https://github.com/davidrunger/david_runger/pull/4810/files?w=1